lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -16,7 +16,15 @@ class WP_Webfonts {
 	 * @access private
 	 * @var array
 	 */
-	private $webfonts = array();
+	private $registered_webfonts = array();
+
+	/**
+	 * An array of enqueued webfonts.
+	 *
+	 * @access private
+	 * @var array
```

Comment:
```suggestion
	 * @var array[]
```

Core doesn't use @access to identify properties visibility. This annotation can be removed. Also this property is an array of arrays when populated as denoted by `array[]`.